motor/discovery/gcp/mql_asset_objects.go

```diff
@@ -38,9 +45,92 @@ func getTitleFamily(o gcpObject) (gcpObjectPlatformInfo, error) {
 	return gcpObjectPlatformInfo{}, errors.Newf("missing runtime info for gcp object service %s type %s", o.service, o.objectType)
 }
 
-func computeImages(m *MqlDiscovery, project string, tc *providers.Config) []*asset.Asset {
+func computeInstances(m *MqlDiscovery, project string, tc *providers.Config, sfn common.QuerySecretFn) ([]*asset.Asset, error) {
 	assets := []*asset.Asset{}
-	images := m.GetList("return gcp.project.compute.images { id name labels }")
+
+	instances, err := m.GetList("return gcp.project.compute.instances { id name labels zone { name } status networkInterfaces disks { guestOsFeatures } }")
```

Comment:
i think you could filter by status in the mql query, to only have to process the running ones

Reply:
That's a good idea. I'll do that.. completely forgot about the power of MQL